.github/CODEOWNERS

```diff
@@ -0,0 +1,24 @@
+# CODEOWNERS file for aspire.dev
+# This file defines who will be requested for review when a pull request modifies certain files.
+# More info: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# Default owner for everything in the repo
+* @IEvangelist
+
+# Index.mdx files
+**/index.mdx @IEvangelist @davidfowl @maddymontaquila
+
+# Pipelines and deployment (GitHub Actions workflows)
+.github/workflows/ @IEvangelist @captainsafia
+
+# Dashboard content
+/src/frontend/src/content/docs/dashboard/ @IEvangelist @JamesNK
+
+# Integrations content
+/src/frontend/src/content/docs/integrations/ @IEvangelist @eerhardt
+
+# Community content (community-toolkit)
+/src/frontend/src/content/docs/community/ @IEvangelist @aaronpowell
```

Comment:
This is fine, but please also add entries for each integration under the integrations directory that has the `⭐ Community Toolkit` badge too.